Configuration/HLT/python/autoCondHLT.py

```diff
@@ -54,4 +64,14 @@ def autoCondHLT(autoCond):
            autoCond[key] = ( autoCond[val[0]] )
         else:
            autoCond[key] = ( autoCond[val[0]],) + val[1]
+#
+    for key in ('run2_mc', 'run2_data', 'run2_data_relval', 'run2_hlt', 'run2_hlt_relval'):
+        old = autoCond[key]
+        autoCond[key] = ( old, ) + l1Menus['GRun']
+
+    for key in ('run2_mc_hi', 'run2_hlt_hi'):
+        old = autoCond[key]
+        autoCond[key] = ( old, ) + l1Menus['HIon']
+
```

Comment:
@mmusich 
Obviously this will have to be removed as soon as you've add the respective L1T menus to these GTs
and before this PR gets integrated; this is just a hack for now to let the PR testing proceed further!